src/components/internalSettings/AccordionSection.tsx

```diff
@@ -4,30 +4,58 @@ import {
   AccordionItem,
   AccordionPanel,
   Box,
-  Button,
-  Input,
   InputGroup,
-  Select,
+  useToast,
 } from "@chakra-ui/react";
-import { useForm } from "react-hook-form";
 import React from "react";
 import axios from "axios";
 
-import { Branch } from "./InternalSettings";
+import { Branch, BranchType } from "./InternalSettings";
+import FormModal from "./FormModal";
 
-const AccordionSection: React.FC<Branch> = props => {
-  const {
-    register,
-    handleSubmit,
-    formState: { isSubmitting },
-  } = useForm();
+export interface FormModalType {
+  _id?: string;
+  name: string;
+  hexathon: string;
+  type: BranchType;
+  settings: {
+    open: string;
+    close: string;
+  };
+  functionName: string;
+  buttonName: string;
+  // eslint-disable-next-line @typescript-eslint/ban-types
+  function: any;
```

Comment:
Can you change the `functionName` and `function` props to something that describes more what the function does? You shouldn't need to use `eslint` to ignore this.